### PR TITLE
[15.0][ADD] product_supplierinfo_comment: New module

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
@sergio-teruel is this still WIP?

**Comment by pedrobaeza** [comment]
@sergio-teruel

**Review by bofiltd — APPROVED** [review]
LGTM

**Comment by pedrobaeza** [comment]
@OCA/board @etobella @sbidoul @dreispt @simahawk this person is approving massively pull requests without real reviews (it has approved about 20 in about 10 minutes). Not sure what are their intentions, but please discard and communicate about this abuse. If trying to scale in the "review ranking", it should be discarded.

**Comment by dreispt** [comment]
Hello @bofiltd , indeed reviews should effectively either be code review or functional testing. Can you confirm this has been your approach?

File: product_supplierinfo_comment/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Product Supplier Info Comment",
+    "summary": """
+        Add the comment field to the product supplier info
+        """,
+    "version": "15.0.1.0.0",
+    "license": "LGPL-3",
```
Comment (pedrobaeza) [inline product_supplierinfo_comment/__manifest__.py]: The header says `AGPL` but this puts `LGPL`. I think it's a mistake.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by sergio-teruel** [comment]
Changes done!

**Review by pedrobaeza — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**